src/pdfviewer/pdfrendermanager.cpp

```diff
@@ -25,6 +25,20 @@ const int kMaxPageZoom = 1000000;
 const qreal kMaxDpiForFullPage = 1000.0;
 
 
+SetImageForwarder::SetImageForwarder(QObject *parent, QObject *obj, const char *rec, QPixmap img, int pageNr):
+    QObject(parent), obj(obj), rec(rec), img(img), pageNr(pageNr)
+{
+}
+
+void SetImageForwarder::setImage() {
+    QMetaObject::invokeMethod(obj, rec, Q_ARG(QPixmap, img), Q_ARG(int, pageNr + 100));
```

Comment:
where is the pageNr+100 coming from ?

Reply:
Ooops. Sorry. That's from debugging session. Will fix now.